webrtc.html

```diff
@@ -6953,7 +6953,19 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       about the underlying transport and the security added.
       <code><a>RTCDtlsTransport</a></code> objects are constructed as a result
       of calls to <code>setLocalDescription()</code> and
-      <code>setRemoteDescription()</code>.</p>
+      <code>setRemoteDescription()</code>. Each
+      <code><a>RTCDtlsTransport</a></code> object represents the DTLS transport
+      layer for the RTP or RTCP <code><a data-link-for="RTCIceTransport">component</a></code>
+      of a specific <code><a>RTCRtpTransceiver</a></code>, or a group of
+      <code><a>RTCRtpTransceiver</a></code>s if such a group has been
+      negotiated via [[!BUNDLE]].</p>
```

Comment:
My intent here was for the prose above to definitively describe the scope of the objects, linking it to SDP (by linking it to transceivers). Where the note below explains the possibly-not-obvious implication of this definition.

Reply:
I think more detail is needed (e.g. whether the objects are created on application of an Offer or an Answer).

Reply:
I'd hoped that https://github.com/w3c/webrtc-pc/pull/1623 would do this, by specifying exactly when the objects should be created (and by omission, when they shouldn't be).

But if we need a note to mention that addTransceiver doesn't create the objects, should that go in the addTransceiver description itself, or here?